cmd/tesla-keygen/main.go

```diff
@@ -90,7 +104,9 @@ func main() {
 	config.RegisterCommandLineFlags()
 	flag.Usage = cliUsage
 	flag.BoolVar(&overwrite, "f", false, "Overwrite existing key if it exists")
+	flag.StringVar(&outputFile, "output", "", "Path to save the public key")
```

Comment:
The output of `tesla-keygen --help` becomes:

```
  -keyring-type type
    	Keyring type (file|keychain|pass). Defaults to $TESLA_KEYRING_TYPE.
  -output string
    	Path to save the public key
  -session-cache file
    	Load session info cache from file. Defaults to $TESLA_CACHE_FILE.
```

To make the argument of `-outupt` "file" rather than "string", which is more consistent with the other options, please change the help string to:

```
Save public key to `file`. Defaults to stdout.
```

Reply:
Suggestion applied